sched/sched/sched_note.c

```diff
@@ -73,41 +127,37 @@ static void note_add(FAR const uint8_t *note, uint8_t notelen);
  * Private Data
  ****************************************************************************/
 
-static struct note_info_s g_note_info;
+#ifdef CONFIG_SCHED_INSTRUMENTATION_FILTER
+static struct note_filter_s g_note_filter;
```

Comment:
The default value will lose all trace data utils the tools from userspace make the correct setting. The initial trace data is very important to debug the bootup issue. Should we inverse the field meaning to enable all trace by default?

Reply:
I understood the situation you want to use.
How about preparing new kernel configuration such as SCHED_INSTRUMENTATION_FILTER_DEFAULT ?